packages/ui/src/services/color-config/color-css-helpers-updater.ts

```diff
@@ -0,0 +1,54 @@
+import { watch } from 'vue'
+import { useGlobalConfig } from '../global-config/global-config'
+import { isServer } from '../../utils/ssr-utils'
+import { addOrUpdateStyleElement } from '../dom-functions'
+import type { ColorConfig } from './color-config'
+
+export type HelperConfig = {
+  stylePrefix: string;
+  styleProperty: string;
+}
+
+export const helperConfigValues: HelperConfig[] = [
```

Comment:
This thing looks pretty useful! 😊

This can be actually moved to GlobalConfig or different plugin as optional service later. 

Can we have some fabric of CSS Helpers? I mean, can user make my own helpers, like `.botder-bottom--primary`?

---


`text--${color}` I think also must care about `fill` CSS property - this will make SVG icons change their colors as well.

Reply:
Yeah, the idea of this type was to provide some functionality to create own utlilty first classes.